### ISSUE TITLE
Validate and Fine-Tune the Model

### ISSUE THREAD

**Issue by tshepang-p** [Mustard-Core/Career-Route-AI#19]
Evaluate the accuracy score and do some editing where necessary

**Comment by tshepang-p** [comment]
@Mzwakhe-H  and @Kgotlello943, your responsibility is to validate and fine-tune the model, please finish with it as soon as possible.

**Comment by Kgotlello943** [comment]
Before running the tunedmodel install scikit-learn and joblib

**Comment by Kgotlello943** [comment]
[tunedmodel.py](https://github.com/user-attachments/files/22261603/tunedmodel.py)
